docs/paper/admin/reference/configuration/bukkit-configuration.mdx

```diff
@@ -0,0 +1,122 @@
+---
+slug: /reference/bukkit-configuration
+---
+
+# Bukkit Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for `bukkit.yml`
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config data={`
+aliases:
+  default: 'now-in-commands.yml'
+  description: 'Static value from converting old versions to the new commands.yml format.'
+chunk-gc:
+  period-in-ticks:
+    default: '600'
+    description: 'How long chunks loaded by plugins should last for.
+      Capped by paper to be 20 ticks (1 second).'
+settings:
+  allow-end:
+    default: 'true'
+    description: 'Whether to load end-type dimensions.'
+  connection-throttle:
+    default: '4000'
+    description: 'How long of a delay to enforce between connections from an IP address.
+      Measured in Milliseconds since last attempt.'
+  deprecated-verbose:
+    default: 'default'
+    description: 'Whether to warn for use of deprecated events.
+      If "default", the warning is printed if annotated with \`@Warning(true)\`.
+      Ignored if [system property paper.alwaysPrintWarningState](/paper/reference/system-properties#paperalwaysprintwarningstate) is true.
+
+      Valid values are "true", "false", or "default".'
+  minimum-api:
+    default: 'none'
+    description: 'Minimum plugin api-version. If below this, or not specified, the plugin is prevented from loading.'
+  permissions-file:
+    default: 'permissions.yml'
+    description: 'File to load Server permissions from.'
+  plugin-profiling:
+    default: 'false'
+    description: 'This option does not operate, as it is disabled by a paper patch.'
+  query-plugins:
+    default: 'true'
+    description: 'Whether to send plugins in the GS4 Query protocol response.'
+  shutdown-message:
+    default: 'Server closed'
+    description: 'The Kick message for the player when the server shuts down.
+
```

Comment:
empty line?

Reply:
Needed for markdown formatting

Reply:
expected so wrapping is skipped imo. can still remove if  you want
![image](https://github.com/PaperMC/docs/assets/42748092/cf3e7da1-a8a1-4458-9bcc-ac77c5907827)